WultraMobileTokenSDKTests/IntegrationUtils.swift

```diff
@@ -0,0 +1,333 @@
+//
+// Copyright 2020 Wultra s.r.o.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions
+// and limitations under the License.
+//
+
+import PowerAuth2
+import WultraMobileTokenSDK
+
+class IntegrationUtils {
+    
+    private static let serverAddress = ProcessInfo.processInfo.environment["SERVER_IP"] ?? "limeserver.pub"
+    private static let paUrl = "http://\(serverAddress):8081/powerauth-java-server"
+    private static let nextStepUrl = "http://\(serverAddress):13010/powerauth-nextstep"
+    private static let enrollmentUrl = "http://\(serverAddress):8080/powerauth-webflow"
+    private static let operationsUrl = "http://\(serverAddress ):8080/powerauth-webflow"
+    private static let nextstepUrl = "http://\(serverAddress):13010/powerauth-nextstep"
```

Comment:
I think that this is insufficient for a proper config. For example, If I would like to test your code against my local server, then I have to change this piece of source code. The problem is that my local Tomcat installation has everything on port `8080`. 

Btw, what is a difference between `nextStepUrl` and `nextstepUrl` constant? I think that the 2nd one is just a typo and is redundant.

Reply:
Yes, fixed.